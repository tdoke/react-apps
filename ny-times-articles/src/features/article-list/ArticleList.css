.article-list-container {
  background-color: #FAFAFA;
}