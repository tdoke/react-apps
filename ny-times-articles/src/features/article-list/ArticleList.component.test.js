import React from 'react';
import { shallow } from 'enzyme';
import ArticleList from './ArticleList.component';

describe('ArticleList', () => {
  let props;
  beforeEach(() => {
    props = {
      articleList: {},
      fetchArticles: jest.fn()
    }
  });

  it('should render ArticleList', () => {
    const component = shallow(<ArticleList {...props} />);
    expect(component).toMatchSnapshot();
  });

});

