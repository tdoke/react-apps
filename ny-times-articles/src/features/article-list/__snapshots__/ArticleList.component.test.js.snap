// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleList should render ArticleList 1`] = `ShallowWrapper {}`;
